indra/newview/llgltfmateriallist.cpp

```diff
@@ -287,15 +293,21 @@ class LLGLTFMaterialOverrideDispatchHandler : public LLDispatchHandler
                         }
                         else if (obj && obj->isAnySelected())
                         {
-                            LLMaterialEditor::updateLive(object_override.mObjectId, results[i].mSide);
+                            for (auto& override_update_callback : callbacks)
```

Comment:
this probably should be a function, like makeCallbacks(object_override.mObjectId, results[i].mSide).